src/openforms/setup.py

```diff
@@ -47,10 +48,14 @@ def load_self_signed_certs() -> None:
     if _certs_initialized:
         return
 
-    # create target directory for resulting combined certificate file
-    target_dir = tempfile.mkdtemp()
-    _load_self_signed_certs(target_dir)
-    _certs_initialized = True
+    lock = RedisLock("load_self_signed_certs")
```

Comment:
This will definitely require some more arguments/options to connect to the correct redis host, especially in container environments. You can take a look at `settings.CACHES` in `base.py` to see how we fetch the redis host information.